README.md

```diff
@@ -239,6 +239,13 @@ There are many places in the format for compatible extensions:
 - Encodings: Encodings are specified by enum and more can be added in the future.
 - Page types: Additional page types can be added and safely skipped.
 
+## Compatibility
+Because of the many features got into the Parquet format it is hard for the
+different implementations to keep up. We introduced the list of "core
```

Comment:
Is it hard? I don't think it's our place to say. We should stay objective and state that some implementations have not kept up.